src/client/tweetAnalysis.js

```diff
@@ -32,12 +74,17 @@ export const mostActiveUsers = (tweets) => {
  * @param tweets An array of Tweet objects
  * @returns {{}} e.g. {'hello': 5, 'world': 8}
  */
-function countWords(tweets) {
+function countWords(tweets, stopList = []) {
   const wordCount = {};
   for (const tweet of tweets) {
-    for (const word of tweet.content.split(/[ !,.?"']/)) {
+    for (const word of tweet.content.split(/[ !,.?"'=]/)) {
       const trimmed = word.trim();
-      if (trimmed !== '') {
+      const lowerCase = trimmed.toLowerCase();
+      if (
+        (lowerCase !== '') &&
+        (stopList.indexOf(lowerCase) == -1) &&

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2195105034/issues/source?bid=3148782&fileBranchId=3167298#l85)